<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgGraphs</class>
 <widget class="QDialog" name="CQDlgGraphs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="qqOldPart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>411</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Data stream recording list</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QListWidget" name="qqRecordingList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Data transformations</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Value = </string>
     </property>
    </widget>
    <widget class="QComboBox" name="qqTransformationCombo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>25</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>25</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqTransformationCoeff">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>25</y>
       <width>66</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>25</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqTransformationOffset">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>25</y>
       <width>66</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Moving average period</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqMovingAveragePeriod">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>45</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Time graph properties</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="qqTimeGraphVisible">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Visible</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqShowLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>25</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Show label</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqLinkPoints">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>25</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Link points</string>
     </property>
    </widget>
    <widget class="QToolButton" name="qqAdjustCurveColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>180</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Adjust curve color</string>
     </property>
    </widget>
    <widget class="QToolButton" name="qqDuplicateToStatic">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>45</y>
       <width>180</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Duplicate curve to static</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="qqAddNewDataStream">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Add new data stream to record</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="qqOldPart2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>XY graphs / 3D curves</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqEditXYGraphs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Edit XY graphs</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqEdit3DCurves">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>25</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Edit 3D curves</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Object size [m]</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqBufferIsCyclic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Buffer is cyclic</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqObjectSize">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>10</y>
     <width>76</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="qqAdjustGridColor">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>75</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Adjust grid color</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqBufferSize">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>30</y>
     <width>76</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="qqRemoveAllStatics">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Remove all static streams/curves</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqExplicitHandling">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Explicit handling</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Buffer size</string>
   </property>
  </widget>
  <widget class="QToolButton" name="qqAdjustBackgroundColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Adjust background color</string>
   </property>
  </widget>
  <widget class="QToolButton" name="qqRemoveAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Remove all streams/curves</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqBufferIsCyclic</tabstop>
  <tabstop>qqObjectSize</tabstop>
  <tabstop>qqExplicitHandling</tabstop>
  <tabstop>qqBufferSize</tabstop>
  <tabstop>qqRemoveAllStatics</tabstop>
  <tabstop>qqAdjustBackgroundColor</tabstop>
  <tabstop>qqAdjustGridColor</tabstop>
  <tabstop>qqAddNewDataStream</tabstop>
  <tabstop>qqRecordingList</tabstop>
  <tabstop>qqTransformationCombo</tabstop>
  <tabstop>qqTransformationCoeff</tabstop>
  <tabstop>qqTransformationOffset</tabstop>
  <tabstop>qqMovingAveragePeriod</tabstop>
  <tabstop>qqTimeGraphVisible</tabstop>
  <tabstop>qqShowLabel</tabstop>
  <tabstop>qqLinkPoints</tabstop>
  <tabstop>qqAdjustCurveColor</tabstop>
  <tabstop>qqDuplicateToStatic</tabstop>
  <tabstop>qqEditXYGraphs</tabstop>
  <tabstop>qqEdit3DCurves</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
